packages/runtime/runtime-utils/src/test/types/validateRuntimeUtilsPrevious.generated.ts

```diff
@@ -264,26 +264,14 @@ use_old_FunctionDeclaration_addSummarizeResultToSummary(
 /*
 * Validate forward compat by using old type in place of current type
 * If breaking change required, add in package.json under typeValidation.broken:
-* "FunctionDeclaration_addTreeToSummary": {"forwardCompat": false}
+* "RemovedFunctionDeclaration_addTreeToSummary": {"forwardCompat": false}
```

Comment:
@anthony-murphy would it be ok to exclude `@internal` things from the type tests?  I can open a task if so.